packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -81,6 +81,10 @@ class DatadogTracer extends Tracer {
   }
 
   _inject (spanContext, format, carrier) {
+    if (!spanContext) {
```

Comment:
Even when there is no active span context, we still want to inject a few properties in the logs like the service name for example. It seems like the problem is that the log propagator supports the absence of a span context, but not the priority sampler. Can this check be done in there instead to return early in the sampler?

Reply:
Done